include/librealsense2/h/rs_option.h

```diff
@@ -104,6 +104,7 @@ extern "C" {
         RS2_OPTION_SEQUENCE_ID, /**< HDR Sequence ID - 0 is not HDR; sequence ID for HDR configuration starts from 1 */
         RS2_OPTION_HUMIDITY_TEMPERATURE, /**< Humidity temperature [Deg Celsius]*/
         RS2_OPTION_ENABLE_MAX_USABLE_RANGE, /**< Turn on/off the maximum usable depth sensor range given the amount of ambient light in the scene */
+        RS2_OPTION_ENABLE_ALTERNATE_IR,  /**< Turn on/off the alternate IR, When enabling alternate IR, the IR image is holding the amplitude of the depth correlation. */
```

Comment:
But they asked for "Alternate IR"...

Reply:
We already already have this option for D400: RS2_OPTION_EMITTER_ON_OFF.

Reply:
@ev-mp What does it do? Here they specifically asked for "Alternate IR" which is not a turning on/off of the emitter. It's different data posing as IR...